changelog.d/sc-549.fixed

```diff
@@ -0,0 +1 @@
+Fixed bug where SSC would fail to generate unreachable findings for a particular depencency if a reachable finding for that dependency had already been found in the same lockfile
```

Comment:
```suggestion
Fixed bug where SSC would fail to generate unreachable findings for a particular dependency if a reachable finding for that dependency had already been found in the same lockfile
```

Reply:
could you add a sentence explaining when you'd want the fixed behavior? I wanted to ask that to understand the point of this PR and figured you might as well explain via the changelog instead of with just a comment :D